extensions/userinfo.py

```diff
@@ -48,13 +48,37 @@ def has_rank_changed(cursor, user_id):
             ORDER BY count DESC
             LIMIT ?
         )
-        SELECT lead FROM leads WHERE user = ?""",
-        (int(os.getenv("RANK_CHANGE_FLOOR", "30")), user_id));
+        SELECT lead FROM leads WHERE user = ?;""",
+        (int(os.getenv("RANK_CHANGE_FLOOR", "40")), user_id))
     row = cursor.fetchone()
     if row is None:
         return False
     return row[0] == 1
 
+def get_user_overtaken(cursor, user_id):
```

Comment:
Is there really no way to combine this with the other query that checks if an overtake has occurred?

Reply:
Maybe? I didn't write these functions